cmd/util/ledger/migrations/cadence_value_diff_test.go

```diff
@@ -174,6 +175,7 @@ func TestDiffCadenceValues(t *testing.T) {
 				address,
 				[]*ledger.Payload{accountStatusPayload},
 				util.RuntimeInterfaceConfig{},
+				snapshot.IndexMapBased,
```

Comment:
nit: maybe use `snapshot.MapBased` for cadence value diff since it is read-only snapshot.

Same comment for other snapshot types in this file.

```suggestion
				snapshot.MapBased,
```